noxfile.py

```diff
@@ -152,6 +155,72 @@ def lint(session: nox.Session) -> None:
     mypy(session)
 
 
+# TODO: node support is not tested yet - it should work if you require('xmlhttprequest') before
+# loading pyodide, but there is currently no nice way to do this with pytest-pyodide
+# because you can't override the test runner properties -
+# https://github.com/pyodide/pytest-pyodide/issues/118
+@nox.session(python="3.11")
```

Comment:
Do we have to restrict this to Python 3.11? Or will any Python 3 do? If there's a range that we can't allow can we filter that out within the task itself?

Reply:
It needs a python supported by `pytest-pyodide` which right now is 3.11